Modules/_testmultiphase.c

```
@@ -952,3 +952,37 @@ PyInit__test_shared_gil_only(void)
 {
     return PyModuleDef_Init(&shared_gil_only_def);
 }
+
+
+#include "datetime.h"
+
+static int
+datetime_capi_client_exec(PyObject *m)
+{
+    _PyDateTimeAPI_Clear();
+    if (PyDateTimeAPI != NULL) {
+        return -1;
+    }
+    PyDateTime_IMPORT;
+    PyErr_Clear();
```

You should not ignore exceptions, but return -1 on error.